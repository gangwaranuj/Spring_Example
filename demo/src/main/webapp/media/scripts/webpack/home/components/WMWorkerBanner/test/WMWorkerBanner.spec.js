import React from 'react';
import { shallow } from 'enzyme';
import WMWorkerBanner from '../index';

describe('<WMWorkerBanner />', () => {
	describe('Rendering ::', () => {
		it('renders correctly', () => {
			const component = shallow(
				<WMWorkerBanner
					invitationsCount={ 14 }
				/>
			);
			expect(component).toMatchSnapshot();
		});
	});
});
