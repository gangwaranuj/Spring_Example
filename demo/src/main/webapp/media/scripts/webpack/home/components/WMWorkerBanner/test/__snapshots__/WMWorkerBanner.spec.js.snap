// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WMWorkerBanner /> Rendering :: renders correctly 1`] = `
<div
  style={
    Object {
      "marginTop": "-1em",
      "position": "absolute",
      "width": "100%",
      "zIndex": "2",
    }
  }
>
  <WMMessageBanner
    hideDismiss={false}
    status="notice"
  >
    You have 
    14
     pending talent pool invitations.
    <WMFlatButton
      href="/groups/invitations"
      label="View Invitations"
      secondary={true}
      style={
        Object {
          "margin": "1em",
        }
      }
    />
  </WMMessageBanner>
</div>
`;
